CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Changed
 * Upgraded Boss Room to Netcode for GameObjects v2.0.0
 * Upgraded editor version to 6000.0.25f1
+* Replaced Lobby and Relay standalone packages with the Multiplayer Services package and the Sessions framework
```

Comment:
Let's include what version of the Multiplayer Services SDK was added.

Also, PR numbers should be included as part of each message in the changelog. The format (#<PR-number>) is what has been used in the past. There's some examples below!